pom.xml

```diff
@@ -418,12 +418,12 @@
 			<dependency>
 				<groupId>org.codehaus.jackson</groupId>
 				<artifactId>jackson-core-asl</artifactId>
-				<version>1.9.13</version>
+				<version>1.9.13-atlassian-2</version>
```

Comment:
Why atlassian-2?

Reply:
julie use `<version>1.9.13</version>` , its the latest full release version. 

`<version>1.9.13-atlassian-2</version>` is like a beta release kinda

Reply:
@jwnasambu   have you seen the above comments?